packages/grafana-toolkit/src/cli/utils/githubRelease.ts

```diff
@@ -57,6 +58,7 @@ class GitHubRelease {
           'Content-Type': resolveContentType(path.extname(file)),
           'Content-Length': fileStat.size,
         },
+        maxContentLength: fileStat.size * 2 * 1024 * 1024,
```

Comment:
What value does this signify, and why do we need a max size?

Reply:
Otherwise the axios client won't upload anything over 10MB, which wasn't an issue until we started compiling all these go programs, (Oracle is over 100MB with the instant client dependencies). So this basically takes bytes -> KB * MB * 2 (double the size in case of errors) is the max allowable content length to upload, and it guarantees that the file will actually be uploaded.